packages/cookie-handler/stories/index.stories.js

```diff
@@ -0,0 +1,198 @@
+import { storiesOf } from '@storybook/html'
+import { withKnobs, object } from '@storybook/addon-knobs'
+import {
+  initializeDemo,
+  styleSource,
+  resetFeature
+} from '../../../helpers/story'
+
+import { features, eventHub } from '@goldinteractive/js-base'
+
+import CookieHandler from '../src/index'
+import '../src/style.scss'
+
+import SampleDocs from './sample.md'
+import BannerDocs from './banner.md'
+import BoxDocs from './box.md'
+
+const sampleTeaser = `
+<style>
+.sample-teaser{
+  width: 400px;
+  height: 200px;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  visibility: hidden;
+  opacity: 0;
+  background-color: rgb(200,200,200);
+}
+
+.sample-close {
+  position: absolute;
+  top: 10px;
+  right: 10px;
+  border: 0px;
+}
+
+.sample-teaser.-show {
+    visibility: visible;
+    opacity: 1;
+  }
+}
+
+</style>
+<div class="sample-teaser" data-feature="cookie-handler" data-cookie-identifier="sample-teaser">
+  <button class="sample-close" type="button" aria-label="Schliessen" data-dismiss>
+        &times;
+  </button>
+  <h3>Sample Teaser</h3>
+  <p>This is a sample Teaser</p>
+</div>
+`
+
+const BannerMarkup = `
+<div class="ft-cookie-disclaimer -banner-bottom -visibility-default -base-theme" data-feature="cookie-handler" data-cookie-identifier="disclaimer-banner" data-cy="disclaimer">
+<p class="disclaimer-text">disclaimer example message <a href="#">Privacy policy</a></p>
+<button data-disclaimer-confirm data-cy="confirm">Got it</button>
+</div>
+`
+
+const BoxMarkup = `
+<div class="ft-cookie-disclaimer -box-bottom-right -visibility-default -base-theme" data-feature="cookie-handler" data-cookie-identifier="disclaimer-box">
+<p class="disclaimer-text">This website uses cookies, with the aim of ensuring that you have the best possible online experience. In using our website, you are giving your consent to the terms and conditions of our Data Protection Declaration / Privacy Statement.<a href="#">Privacy policy</a></p>
+<button data-disclaimer-confirm>Got it</button>
+</div>
+`
+
+storiesOf('CookieHandler', module)
```

Comment:
These stories contain quite a lot of Copy Paste logic. Maybe it would make sense to refactor this in order to reduce code bloat?